### DISCUSSION TITLE
Tdarr / Proxmox / Handbrake Issue (I think)!

### DISCUSSION THREAD

**Discussion by thearchitectuk** [tteck/Proxmox#2312] [Help Needed]
Hi all, apologies in advance, I'm quite new to Proxmox & Tdarr and have got a little bit stuck!

I've set it all up Tdarr using tteck's script, mounted relevant paths to my synology nas, but as soon as it starts to encode, it bounces everything in the queue.

Is Handbreak automatically installed as part of the Tdarr script or have I missed something this obvious? Thanks

![image](https://github.com/tteck/Proxmox/assets/35996506/e9d11350-5e97-4809-9bf7-5798e37bd525)

**Comment by mike6041** [comment]
Same issue here

**Reply by papafalafel** [reply]
Also have the same issue. Just did a fresh install to see if that would fix it but it didn't l.

**Comment by Narkowii** [comment]
same issue here

**Comment by zoddti** [comment]
I think apt install handbrake-cli solved this for me if i remember correctly.

**Reply by Narkowii** [reply]
i'll try, TY
